src/main/resources/hudson/plugins/groovy/Pipeline.groovy

```diff
@@ -0,0 +1,10 @@
+public class Pipeline {
+    public static Object input() {
+        f('input.ser').withObjectInputStream {it.readObject()}
+    }
+    public static void output(Object o) {
+        f('output.ser').withObjectOutputStream {it.writeObject(o)}
+    }
+    private static File f(String name) {new File(new File(Pipeline.class.getResource('/Pipeline.groovy').toURI()).parent, name)}
```

Comment:
Please document this resolve to the `tmp` directory as the code is nontrivial to follow.

Reply:
agreed